run.py

```diff
@@ -16,6 +19,8 @@ def create_app(config_filename: str) -> Flask:
 
     migrate = Migrate(app, db)
 
+    cors.init_app(app, resources={r"*": {"origins": "http:localhost:5000"}})
```

Comment:
Does BIT required access to all endpoints of the mentorship backend or can we give access to specific endpoints instead of all the endpoints?

Reply:
Yes, @epicadk. With users functionaliities and mentorship relation are managed by Mentorship System, BIT do need to have access to all Mentoship System api endpoints 😉.

Reply:
Do you think we should fetch the origins property based on the build type i.e. production, dev, local etc.?

Reply:
@epicadk, I think it's best to keep it as it for now, coz majority of contributors work on local machine atm. When we have remote servers for dev and production, we can revisit this origin property.